script/deploy/facets/DeployDummy.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
not sure what this file is needed for...!?

Reply:
This was just s test to make sure I could get forge working. The other scripts keep failing because we do a lot of config via ENV vars which is pretty messy. This won't stay in the final version.